contrib/meson.build

```diff
@@ -1,3 +1,4 @@
+subdir('ci')
```

Comment:
I don't think we should have anything in `contrib/ci` as default unit tests.  can this be moved elsewhere?

Reply:
I think it's fine right now - but don't you want the pre-commit hook too to check this?